defe/formatter.py

```diff
@@ -27,8 +28,24 @@ def defy_prompt(feed):
                     input(Fore.GREEN + Style.BRIGHT + "Enter Feed Index to open : ")
                 )
                 print(Style.RESET_ALL)
-                print(Style.BRIGHT + "Opening Link in your default browser ...")
-                webbrowser.open(feed[int(feed_to_open) - 1].link)
+                if podcasts:
+                	player = vlc.MediaPlayer(feed[int(feed_to_open) -1]["links"][0]["href"])
```

Comment:
It would be nice if we inform the user of what is playing right now before we invoke VLC.
Say for e.g we can output it like this

[Now Playing] `<Podcast title>`

Reply:
I have added this feature